docs/userGuide/syntax/thumbnails.mbdf

```diff
@@ -0,0 +1,46 @@
+## Thumbnails
+
+**A `thumbnail` component allows you to insert thumbnails using text, images, or icons.**
+
+<include src="outputBox.md" boilerplate >
+<span id="code">
+
+```html
+<thumbnail circle src="{{baseUrl}}/images/deer.jpg" size="150"/>
+<thumbnail circle text=":book:" background="#333" fontcolor="white" size="100"/>
+<thumbnail circle text="___CS___" background="#ccf" fontcolor="#339" border="3px solid #339" size="100"/>
+<thumbnail text=":fas-book:" fontcolor="darkgreen" size="100"/>
+```
+</span>
+<span id="output">
+<thumbnail circle src="{{baseUrl}}/images/deer.jpg" size="150"/>
+<thumbnail circle text=":book:" background="#333" fontcolor="white" size="100"/>
+<thumbnail circle text="___CS___" background="#ccf" fontcolor="#339" border="3px solid #339" size="100"/>
+<thumbnail text=":fas-book:" fontcolor="darkgreen" size="100"/>
+</span>
+</include>
+
+****Options****
+Name | Type | Default | Description
+--- | --- | --- | ---
+alt | `string` | | The browser displays this if the image fails to load
```

Comment:
Forgot to add **This must be specified.** 

Description of shared options between `<pic>` and `<thumbnail>` should be standardised. Thus you should use `The alternative text of the image.` as the description.

Reply:
Ok